lib/index.js

```diff
@@ -1,60 +1,133 @@
-        const childArtist = document.createElement('h4');
-        const appendChildArtist = parentArtist.appendChild(childArtist);
-        appendChildArtist.innerHTML = json.message.body.track_list[i].track.artist_name;
+      const playlistCount = json.length
 
-        const parentSong = document.getElementById('search_results');
-        const childSong = document.createElement('h4');
+      for (let i = 0; i < playlistCount; i++) {
+        const parentSong = document.getElementById('all_playlists');
+        const childSong = document.createElement('h3');
         const appendChildSong = parentSong.appendChild(childSong);
-        appendChildSong.innerHTML = json.message.body.track_list[i].track.track_name;
-
-        const parentAlbum = document.getElementById('search_results');
-        const childAlbum = document.createElement('h4');
-        const appendChildAlbum = parentAlbum.appendChild(childAlbum);
-        appendChildAlbum.innerHTML = json.message.body.track_list[i].track.album_name;
+        appendChildSong.innerHTML = "Playlist Name: " + json[i].playlist_name;
       }
     })
     .catch(error => console.log(error));
-  };
+};
+
+function getArtist(event) {
+  event.preventDefault();
+  const artist = document.getElementById('search_field').value;
+
+  fetch(`https://cors-anywhere.herokuapp.com/http://api.musixmatch.com/ws/1.1/track.search?apikey=b24629931e9a581edb617d6b39e5563f&q_artist=${artist}&s_track_rating=desc`)
+  .then(result => {
+    return result.json();
+  })
+  .then(json => {
+    document.getElementById('search_results').style.display = '';
+    document.getElementById('all_favorites').style.display = 'none';
+    document.getElementById('all_playlists').style.display = 'none';
+
+    const songCount = json.message.body.track_list.length
+
+    for (let i = 0; i < songCount; i++) {
+      const parentArtist = document.getElementById('search_results');
+      const childArtist = document.createElement('h3');
+      const appendChildArtist = parentArtist.appendChild(childArtist);
+      appendChildArtist.innerHTML = "Artist: " + json.message.body.track_list[i].track.artist_name;
+
+      const parentSong = document.getElementById('search_results');
+      const childSong = document.createElement('h4');
+      const appendChildSong = parentSong.appendChild(childSong);
+      appendChildSong.innerHTML = "Song: " + json.message.body.track_list[i].track.track_name;
+
+      const parentAlbum = document.getElementById('search_results');
+      const childAlbum = document.createElement('h6');
+      const appendChildAlbum = parentAlbum.appendChild(childAlbum);
+      appendChildAlbum.innerHTML = "Album: " + json.message.body.track_list[i].track.album_name+ `</br> <button id="add_favorite_${i}" class="btn btn-lg btn-success" type="text">Add to Favorites</button>`;
+    }
+  })
+  .catch(error => console.log(error));
+};
+
+// function createFavorite(event) {
```

Comment:
we will probably want to clean this up later when we do a scrub of comments and notes from the code.